src/command_modules/azure-cli-monitor/HISTORY.rst

```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+unreleased
++++++++++++++++++++++
+
+* Update azure-mgmt-monitor to version 0.2.0 and azure-monitor to version 0.3.0
```

Comment:
This comment should be more customer facing. They don't care about version bumps--they care what functionality was added or what bugs were fixed. Based on how small the PR is--I wouldn't add anything to this section.